AlertaDengue/dados/views.py

```diff
@@ -1082,7 +1084,7 @@ def view_filter_city(self, context):
         self.template_name = 'report_filter_city.html'
 
         options_cities = ''
-        for geocode, city_name in dbdata.get_cities(
+        for geocode, city_name in RegionalParameters.get_cities(
```

Comment:
Maybe is is a good Idea to cache this call.

Reply:
#### Here it is used only to list the cities by state:
view_filter_city():
![filter_report](https://user-images.githubusercontent.com/3450741/141184921-284926f5-e97e-4791-a9be-4d3b7433c1bb.png)